examples/audio-classification/README.md

```diff
@@ -107,52 +107,6 @@ On 8 HPUs, this script should run in ~12 minutes and yield an accuracy of **80.4
 
 > If you get an error reporting unused parameters in the model, you can specify `--ddp_find_unused_parameters True`. Using this parameter might affect the training speed.
 
-
```

Comment:
Removing deepspeed example here, since the above multi HPUs with torch compile shows much better performance(with same memory usage) then deepspeed command